internal/pkg/describe/status_describe.go

```diff
@@ -156,13 +160,29 @@ func (s *ecsStatusDescriber) Describe() (HumanJSONStringer, error) {
 	if err != nil {
 		return nil, fmt.Errorf("get tagged CloudWatch alarms: %w", err)
 	}
-	alarms = append(alarms, taggedAlarms...)
+	for _, alarm := range taggedAlarms {
+		alarms[alarm.Name] = alarm
+	}
 	autoscalingAlarms, err := s.ecsServiceAutoscalingAlarms(svcDesc.ClusterName, svcDesc.Name)
 	if err != nil {
 		return nil, err
 	}
-	alarms = append(alarms, autoscalingAlarms...)
-
+	for _, alarm := range autoscalingAlarms {
+		alarms[alarm.Name] = alarm
+	}
+	rollbackAlarms, err := s.ecsServiceRollbackAlarms(s.app, s.env, s.svc)
+	if err != nil {
+		return nil, err
+	}
+	for _, alarm := range rollbackAlarms {
+		alarms[alarm.Name] = alarm
+	}
+	alarmList := make([]cloudwatch.AlarmStatus, len(alarms))
+	i := 0
```

Comment:
nit: ⬇️ should initialize `i` as `0`
```suggestion
	var i int
```